web/shared/api.ts

```diff
@@ -13,22 +13,36 @@ export const removeUnauthorizedCallback = authMiddleware.removeUnauthorizedCallb
 export function deleteSession(streamId: string, clientId: string) {
     return w.url(`/session/${streamId}/${clientId}`).delete().res();
 }
+export function getCurrentNode(): string {
+    const urlParams = new URLSearchParams(window.location.search);
+    return urlParams.get('nodes') || '0';
+}
 
-export function createStream(streamId: string) {
-    return w.url(`/api/streams/${streamId}`).post().res();
+export async function createStream(streamId: string): Promise<unknown> {
+    const currentNode = getCurrentNode();
+    try {
+        return await w.url(`/api/streams/${streamId}?nodes=${currentNode}`).post().res();
```

Comment:
This is Liveman UI and Live777 UI common function, Live777 dispatcher is correct, but liveman need random dispatcher, There has problem: only dispatch `nodes=0`